Search/search.tex

```diff
@@ -103,7 +103,7 @@ \section{Query Entries and Tags}
 the data tagged as \code{pcap} is binary (actual packets) it can't really
 be printed as text, thus the ``noise''. Later in this chapter, we will show how to
 process this binary data.
-
+\pagebreak
```

Comment:
why force a page break here? The following image goes with this section. If anything, I'd suggest a \clearpage *after* the following figure

Reply:
I fetched the PR and tried it with a \clearpage after the figure and it looked good... if you don't mind making that change, I'm OK with the rest.